packages/server/src/index.ts

```diff
@@ -1,67 +1,66 @@
+import axios from 'axios'
```

Comment:
are you using smtg like formatter that will rearrange the imports by alphabetical order?

Reply:
It's a VS code extension. If it's causing trouble, I can manually change them back.

Reply:
ah okay, if you can remove the changes that are not part of the actualy web scraping changes, that'd be great!